src/posts/2024-03-20-kubecon-eu/index.md

```diff
@@ -17,12 +17,101 @@ in Paris! We'll keep this page updated with more details as they are published.
 
 - Come to the [Cilium + eBPF Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/cilium-ebpf-day/) co-located event
 
-- Attend the Maintainers' Track session (details coming soon)
+- Attend the Maintainers' Track session
 
-- Get involved with Cilium at the ContribFest (details coming soon)
+  These are the details for our maintainer track session, we hope to see you there!
+
+  [Cilium: Connecting, Observing, and Securing Service Mesh and Beyond with EBPF - Liz Rice & Maartje Eyskens, Isovalent; Nico Meisenzahl, white duck; Vlad Ungureanu, Palantir Technologies](https://kccnceu2024.sched.com/event/1Yhfl/cilium-connecting-observing-and-securing-service-mesh-and-beyond-with-ebpf-liz-rice-maartje-eyskens-isovalent-nico-meisenzahl-white-duck-vlad-ungureanu-palantir-technologies?iframe=no&w=100%&sidebar=yes&bg=no)
+
+  Wednesday March 20, 2024 15:25 - 16:00 CET
+
+  Welcome to Cilium's maintainer track session where you'll get an update on how Cilium is expanding the frontiers of cloud native networking, observability, and security. After graduating and becoming the de-facto standard CNI for Kubernetes, what comes next? We'll start with a brief overview of each part of the project before giving a update on Cilium Service Mesh's mutual authentication. Next, we'll dive into how Cilium is expanding beyond Kubernetes with load balancing and multi-cloud networking and into runtime enforcement with Tetragon. In this session, you'll hear from Cilium contributors and users Isovalent, Palantir, and white duck.
 
 - Come and have a chat at the Cilium kiosk in the Project Pavilion throughout
   the event
 
-Come back to this page for a full program of Cilium-related talks when the
-schedule is announced. À bientôt à Paris!
+## Cilium Talks at KubeCon EU 2024 Full Schedule
+
+Below is a list of some Cilium talks at KubeCon EU including Cilium + eBPF Day, Observability Day and Istio Day.
```

Comment:
Majorly used talks from Cilium + eBPF Day since it's like our major event. I went with talks that covered Cilium, Tetragon, Edge and were from the end users I mentioned in the other blog post. I also tried to capture different people so it's a bit more diverse, so we don't get into any fights.

I included both the opening and closing sessions so people know when the event ends and begins.

Reply:
The title says full schedule, but the list is only a portion. The description also says Istio and Observability day, but I don't see those talks. How about just adding the social card and linking to the blog post with all of the talks?

Reply:
fair enough, would just do that, i was trying to not repeat what we already have in the other blog post

Reply:
done @xmulligan